homeassistant/components/deconz/__init__.py

```diff
@@ -166,6 +166,11 @@
     return True
 
 
+async def async_options_updated(hass, config_entry):
```

Comment:
I am not sure if we should do this like this, or should the flow do the updates themselves when the flow is finished (calls create entry etc)

Reply:
I think just as with ordinary config entries that any changes to a component should be executed after the config flow has finished. If it is to send events or whatever. I like that it follows the same path as setup_entry/unload_entry

Reply:
I agree that we should notify the components, but I wonder if we should go down the path of load/unload as this change does, or we go down the road of entity registry and allow attaching change listeners to a config entry. Because that way, we can re-use that trigger mechanism for other changes to the config entry, and not end up with a method per property of a config entry.

Reply:
Sounds reasonable, looking into this in detail tomorrow.

Reply:
@balloob better? Followed entity registry, do we want to add some kind of information/diff of what has changed?

Reply:
Diffing is difficult as we assign directly.

Reply:
Allright